queue_util/consumer.py

```diff
@@ -149,52 +149,43 @@ def run_forever(self, wait_timeout_seconds=None, **kwargs):
             try:
                 self.wait_if_paused()
 
-                message = self.source_queue.get(block=True, timeout=wait_timeout_seconds)
-                data = message.payload
+                try:
+                    message = self.source_queue.get(block=True, timeout=wait_timeout_seconds)
+                    data = message.payload
+                except queue.Empty:
+                    continue
+                except Exception as e:
+                    logging.exception("Exception getting message from Rabbit", str(e))
```

Comment:
Think this needs a `%s` to work (tested in py3.8):

```suggestion
                    logging.exception("Exception getting message from Rabbit: %s", str(e))
```

Reply:
Oh that's how it works! :facepalm:

Reply:
I totally thought it'd work but wasn't 100% and actually errored out locally!

Reply:
9dcb155